### DISCUSSION TITLE
Argo CD

### DISCUSSION THREAD

**Discussion by gigo1980** [kube-hetzner/terraform-hcloud-kube-hetzner#612] [Q&A]
Hi together,

has anyone installed ArgoCD? I have an issue that the grpc is not working. The ArgoCD CLI uses this. I have running an older Cluster also on hetzner with argocd and there it is working fine.

regards Enrico

**Comment by ianwesleyarmstrong** [comment]
Have you tried using this [ingressRoute](https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#ingressroute-crd)? This should redirect the gRPC requests appropriately.

**Reply by ianwesleyarmstrong** [reply]
Following up, I had some success with this [IngressRoute](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/discussions/615#discussioncomment-5234573)!
